app/src/main/java/org/simple/clinic/ageanddateofbirth/DateFormatterModule.kt

```diff
@@ -13,10 +13,10 @@ class DateFormatterModule {
   @Provides
   @AppScope
   @Named("date_for_search_results")
-  fun provideShortDigitDateFormatter(): DateTimeFormatter = DateTimeFormatter.ofPattern("d/MM/yyyy", Locale.ENGLISH)
+  fun provideDateFormatterForSearchResults(locale: Locale): DateTimeFormatter = DateTimeFormatter.ofPattern("d-MMM-yyyy", locale)
```

Comment:
TODO: Ask why are we changing from `MM` to `MMM`

Reply:
This was already discussed in person, but adding it here for the record.

The earlier "date_for_search_results" was not being used anywhere. The actual date formatted in the `PatientSearchResultsAdapter` was being instantiated in the class, using this format.